assets/svelte/components/SidebarSection.svelte

```diff
@@ -75,7 +76,17 @@
       on:click={() => (expanded = !expanded)}
       aria-expanded={expanded}
     >
-      <span><slot name="heading" /></span>
+      <span>
+        <slot name="heading" />
+        {#if !disableDelete}
```

Comment:
I think it should not display the X icon in the `Classes` attr

Reply:
Removed